edi_storage_oca/components/listener.py

```diff
@@ -14,21 +14,16 @@ class EdiStorageListener(Component):
     def _move_file(self, storage, from_dir_str, to_dir_str, filename):
         from_dir = PurePath(from_dir_str)
         to_dir = PurePath(to_dir_str)
-        if filename not in storage.list_files(from_dir.as_posix()):
-            # The file might have been moved after a previous error.
+        # TODO:
+        # - storage.list_files now includes path in fs_storage, breaking change
+        # - we remove path
+        files = storage.list_files(from_dir.as_posix())
+        files = [os.path.basename(f) for f in files]
+        if filename not in files:
             return False
-        self._add_after_commit_hook(
```

Comment:
Why do you remove this? This is needed to make the deletion happened only if the transaction is successful.

Reply:
Sorry, i have a missing some where, I have undone it